<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="CometWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>761</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Ugao</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sl_scale_1">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>90</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_scale_1">
       <property name="text">
        <string>0</string>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Poƒçetna brzinna</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sl_scale">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_scale">
       <property name="text">
        <string>0</string>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>761</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CometWidget</class>
   <extends>QWidget</extends>
   <header location="global">cometwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sl_scale</sender>
   <signal>valueChanged(int)</signal>
   <receiver>l_scale</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>793</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sl_scale_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>l_scale_1</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
